android/java/org/chromium/chrome/browser/vpn/activities/BraveVpnPlansActivity.java

```diff
@@ -173,23 +175,17 @@ public void showRestoreMenu(boolean shouldShowRestore) {
         invalidateOptionsMenu();
     }
 
-    @Override
-    public void showProgress() {
-        if (mPlanProgress != null) {
-            mPlanProgress.setVisibility(View.VISIBLE);
-        }
-        if (mPlanLayout != null) {
-            mPlanLayout.setAlpha(0.4f);
-        }
-    }
+    // @Override
+    // public void showProgress() {
+    //     BraveVpnUtils.showProgressDialog(
+    //             BraveVpnPlansActivity.this, getResources().getString(R.string.vpn_connect_text));
+    // }
+
+    // @Override
+    // public void hideProgress() {
+    //     BraveVpnUtils.dismissProgressDialog();
+    // }
```

Comment:
does this commented block need to be left in the code?

Reply:
Sorry missed the commented code.